drivers/bno055/include/bno055_internal.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2019 Wouter Horlings
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_bno055
+ *
+ * @{
+ * @file
+ * @brief       Definitions for the bno055 device
+ *
+ * @author      Wouter Horlings <wouter@horlin.gs>
+ */
+
+#ifndef BNO055_INTERNAL_H
+#define BNO055_INTERNAL_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    BNO055 chip id registers and who am I values
+ * @{
+ */
+#define BNO055_REG_CHIPID          (0x00U)  /**> chip id register */
```

Comment:
```suggestion
#define BNO055_REG_CHIPID          (0x00U)  /**< chip id register */
```

This doxygen issue is present all over the place.
In current master:
```
$ git grep "/\*\*>" | wc -l
0
git grep "/\*\*<" | wc -l
373126
```